### ISSUE TITLE
Ignite proccing from Grenades

### ISSUE THREAD

**Issue by Atchooom** [slowtorta/turtlewow-bug-tracker#5310]
Hello,

I am playing a Mage with Ignite talent, the bug is that this talent can proc from Iron Grenade when critting, instantly cancelling the stun effect. Effectively griefing the Fire spec in Pvp, kind of sad when Frost is already so much more encouraged...

**Steps to reproduce the behavior with as much detail as possible:**

1. Iron Grenade an enemy target while having points in Ignite 
2. Repeat until the Grenade crits, hence proccing Ignite and cancelling the stun effect of the grenade

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**

Spell: https://database.turtle-wow.org/?spell=11119      Ignite
Object: https://database.turtle-wow.org/?item=4390      Iron grenade

Expected behaviour :
Grenades should not proc Ignite.

**Comment by twowTurtReynolds** [comment]
I believe sappers are also proccing ignite? And I am not exactly sure if either is blizzlike behavior or not.

**Comment by Atchooom** [comment]
I will pay more attention and try to let you know if they do. TBH it would be kind of a buff if it does, it's a nerf on grenade because it breaks the stun.
But for the sake of integrity of correctly fixing actual bugs, I never saw sappers proccing ignites on Blizz servers (not sure tho that you want them as absolute reference).

**Comment by twowTurtReynolds** [comment]
Indeed, we aim for little mechanics like these to be 1:1 blizzlike, so that players who come from 2006 or classic will feel at home with how the underlining systems of vanilla are all correctly in place.

**Comment by Atchooom** [comment]
Can i do anything more to help fixing this bug ?

I saw this post : https://github.com/slowtorta/turtlewow-bug-tracker/issues/3330
It's from April and the Goblin Mortar issue described in the post still hasnt been fixed.
So i'm wondering if the issue i report will get a fix too.
There is another issue with Portable Bronze Mortar acting like Big Bronze Bomb, but i will keep this for another post.
I'm quite unfamiliar with Github and reporting bugs, so pardon if i seem rude, i just want to play without the bug :3
Thank you for your work

**Comment by twowTurtReynolds** [comment]
This is about it, maybe add a video to your original post. Thank you for our patience. I do hope you can see our 1,000 bug reports open so I'm sorry if this one isn't a high priority and doesnt get resolved asap.

**Comment by Atchooom** [comment]
Alright i will see if i can do that, indeed the page is busy ! Thx again, game is great :)
